src/h3lib/lib/h3Index.c

```diff
@@ -860,6 +885,29 @@ void H3_EXPORT(h3GetFaces)(H3Index h3, int* out) {
     }
 }
 
+/**
+ * pentagonIndexCount returns the number of pentagons (same at any resolution)
+ *
+ * @return int count of pentagon indexes
+ */
+int H3_EXPORT(pentagonIndexCount)() { return 12; }
+
+/**
+ * Generates all pentagons at the specified resolution
+ *
+ * @param res The resolution to produce pentagons at
+ */
+void H3_EXPORT(getPentagonsAtRes)(int res, H3Index* out) {
+    int i = 0;
+    for (int bc = 0; bc < NUM_BASE_CELLS; bc++) {
+        if (_isBaseCellPentagon(bc)) {
```

Comment:
Would it be easier just hard-coding the indexes of the 12 base cells here? I believe they're `4, 14, 24, 38, 49, 58, 63, 72, 83, 97, 107, 117` - we could add a constant in `baseCells.h`. OTOH, I doubt performance is an issue here, so 🤷‍♂

Reply:
I did consider this but felt bad about adding a hard-coded list in addition to `BaseCellData` which already contains the information. What would you prefer?

Reply:
Probably best to leave it as is - still constant time and probably not worth over-optimizing on performance for this function. So nvm :).